dev/README.md

```diff
@@ -86,30 +86,30 @@ The frontend uses a custom Dockerfile with the following modifications:
 
 ```bash
 cd /frontend
-npm start -- --host 0.0.0.0
+npm run start -- --host 0.0.0.0 --disable-host-check
 ```
 
-A custom Dockerfile is used because the production image uses the node alpine base image which does not crosscompile on macOS.
+A custom Dockerfile is used because the production image builds the static site and then serves it via nginx. The development image serves the site using `ng serve` (webpack-dev-server) so it reflects the latest code and updates when files change.
```

Comment:
does the official image work on your mac? Because the alpine version of node did not work on mac for me. If this is the case for you as well, I would  keep the comment as is and also keep the custom image with stages and the docker-file with target

Reply:
nginx:1.2-alpine doesn't work on arm64. Newer versions of nginx do. We should update all our images to use a recent alpine version.